### DISCUSSION TITLE
Restart Conversation Button

### DISCUSSION THREAD

**Discussion by alxspiker** [Mobile-Artificial-Intelligence/maid#52] [Ideas]
Really would love to be able to restart the conversation or test new parameters without having to reload app. And if there is a way to do this, I can't figure it out. Same issue with Sherpa. Also a way to not have a prepompt would be nice so I can use code llama and such without feeding in preset data.

**Comment by danemadsen** [comment]
Currently The parameters refresh every time you open and close either the model page or the character page. Ive just added the ability to branch conversations today so you can use that feature for what you're trying to do, and at some point ill work on a conversation menu for persistent chats.

**Reply by alxspiker** [reply]
That's awesome! That should help a lot, but I was thinking for the conversation history to reset by one click, almost like Bing AI. But I am starting to notice how bad the caching issue is so I don't think it would be even useful to have a restart conversation button.

**Reply by alxspiker** [reply]
Just making a coffee and I'll figure out that caching issue. Might be able to do some weird ADB hack or something for now. I'll test some things out.
